explorer/interpreter/resolve_unformed.cpp

```diff
@@ -171,6 +176,22 @@ static auto ResolveUnformed(Nonnull<const Expression*> expression,
   return Success();
 }
 
+static auto ResolveUnformed(Nonnull<const Expression*> expression,
+                            FlowFacts& flow_facts, FlowFacts::ActionType action)
+    -> ErrorOr<Success> {
+  // We visit subexpressions in evaluation order, by performing a LIFO
+  // traversal here and enqueueing subexpressions in reverse order in
+  // ResolveOneUnformedExpression.
+  llvm::SmallVector<PendingResolveUnformedStep, 32> queue;
```

Comment:
The explorer predominantly uses std types -- I think this would be the first use of SmallVector.  Would it make sense to switch?